lib/MetaCPAN/Document/Author/Set.pm

```diff
@@ -0,0 +1,112 @@
+package MetaCPAN::Document::Author::Set;
+
+use strict;
+use warnings;
+
+use Moose;
```

Comment:
Are these classes able to use `MetaCPAN::Moose` or does the ESX model get in the way?